dts/bindings/i2c/i2c-slave.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018, Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: I2C Slave Base Structure
+id: i2c-slave
```

Comment:
Can this become i2c-device instead of i2c-slave?

Convention in e.g. PR #9876 is to call devices that are attached to a bus a [bus]-device. Adhering to this convention could help in the extraction of device tree properties.

Reply:
Removed this file and switched to using i2c-device